website/static/index.html

```diff
@@ -32,7 +32,8 @@
       <a href="#" class="brand w-nav-brand"><img src="images/nav-logo.png" loading="lazy" alt="" class="nav-logo"></a>
       <nav role="navigation" class="nav-menu w-nav-menu">
         <a href="#" class="nav-link w-nav-link">Home</a>
-        <a href="https://onetable.dev/docs/setup/" class="nav-link w-nav-link">Docs</a><img src="images/break.svg" loading="lazy" alt="" class="image-10">
+        <a href="https://onetable.dev/docs/setup/" class="nav-link w-nav-link">Docs</a>
+        <a href="https://onetable.dev/blog" class="nav-link w-nav-link">Blog</a><img src="images/break.svg" loading="lazy" alt="" class="image-10">
```

Comment:
should it be blog or blogs?

Reply:
just wanted to keep it consistent with the github folder structure. we could call it `Blogs` on the site.